code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm

```diff
@@ -266,7 +270,8 @@ datum/reagent/water/reaction_turf(var/turf/simulated/T, var/volume)
 
 datum/reagent/water/reaction_obj(var/obj/O, var/volume)
 	src = null
-	O.color = initial(O.color)
+	if(!istype(O, /obj/effect))
```

Comment:
I know you're doing this to prevent blobs from getting 'cleaned' but the point is that you can clean ANYTHING that is colored, not just blobs and shit.

It's a big problem that'll only get worse as we make more use of the color var (which we should)